### PR TITLE
Proper closing of scaladoc comments even with quotes

### PR DESCRIPTION
**Pull request by dragos**
Fix #1001650.

### CODE REVIEW

File: org.scala-ide.sdt.core.tests/src/org/scalaide/core/ui/CommentAutoEditStrategyTest.scala [hunk 1]
```diff
@@ -104,6 +104,24 @@ class CommentAutoEditStrategyTest extends AutoEditStrategyTests {
   }
 
   @Test
+  def openDocComment_topLevel_with_stringLitCommon() {
+    """
+    /**
+     * ""^
+    class Foo {
```
Comment (Kwestor) [inline org.scala-ide.sdt.core.tests/src/org/scalaide/core/ui/CommentAutoEditStrategyTest.scala]: I don't know if I understand this test case correctly, but issue states "Closed multi line comment is closed again". 

This test looks like it only checks if it closes correctly the first time.

Comment (dragos) [inline org.scala-ide.sdt.core.tests/src/org/scalaide/core/ui/CommentAutoEditStrategyTest.scala]: Good catch! Will update.

**Comment by dragos** [comment]
@Kwestor I made a few code changes and fixed the test

**Comment by Kwestor** [comment]
LGTM, I'll wait for Jenkins and merge.

**Comment by dragos** [comment]
Thanks!

**Pull request merged**